### ISSUE TITLE
Misalignment between row header and rows when rowHeights set

### ISSUE THREAD

**Issue by jrowen** [handsontable/handsontable#3895]
Using http://jsfiddle.net/handsoncode/s6t768pq/, if I specify the table set-up as shown below, I'm seeing a misalignment between the row header and the rows.
```
var hot = new Handsontable(container, {
  data: data(),
  minSpareCols: 1,
  minSpareRows: 1,
  rowHeaders: true,
  colHeaders: true,
  contextMenu: true,
  rowHeights: 18
});
```
![screenshot from 2016-11-17 09-31-39](https://cloud.githubusercontent.com/assets/1530030/20395523/e3f6d2a6-aca8-11e6-9663-d2028377d6c6.png)

If I specify `rowHeights: [18]` instead, the alignment is better, but still off slightly.
![screenshot from 2016-11-17 09-34-47](https://cloud.githubusercontent.com/assets/1530030/20395629/362e8686-aca9-11e6-943f-275cab593574.png)

**Comment by AMBudnik** [comment]
Hi @jrowen 

Handsontable in default sets `rowHeight` to `22px` and it does not accept lower values so easily that is why when you type `23px` and above headers are looking as expected.

**Comment by jrowen** [comment]
Thanks for the quick response.  To save future confusion, it might be worthwhile adding the default to the [rowHeights documentation](https://docs.handsontable.com/0.29.0/Options.html#rowHeights).  Also, would it make sense to ignore values < 22px?

**Comment by AMBudnik** [comment]
That's right, we should add an information about default settings.

**Comment by AMBudnik** [comment]
Hi @jrowen 

We have just fixed the issue in version 0.33.0. Can you update and check if it works in your project as well?

**Comment by AMBudnik** [comment]
I am closing this issue, however, feel free to leave any additional comments. Thanks in advance!
